backend/src/main/java/com/soen341/instagram/service/impl/CommentService.java

```diff
@@ -133,4 +153,24 @@ public Comment findComment(final long commentId)
 		}
 		return commentOptional.get();
 	}
+
+	public CommentResponseDTO determineEditable(final CommentResponseDTO commentResponseDTO)
+	{
+		String currentUser = null;
+		if (!(SecurityContextHolder.getContext().getAuthentication() instanceof AnonymousAuthenticationToken))
```

Comment:
Not sure what I get how `AnonymousAthenticationToken` works (in this context, at least)....mind explaining why we would want the authentication to be anonymous to set the currentUser?

Reply:
It's the opposite, I want to set the current when they are not anonymous.

Reply:
My bad, definitely didn't see the `!` not modifier in front. That's what I thought was the logical way to go but I guess I skimmed over that part, sorry!